Packs/ProactiveThreatHunting/Layouts/layoutscontainer-Proactive_Threat_Hunting.json

```diff
@@ -0,0 +1,1361 @@
+                                "fieldId": "attackpatterns",
+                                "height": 53,
+                                "id": "26f94520-d1f0-11ed-acc6-b384b0c94da3",
+                                "index": 2,
+                                "listId": "caseinfoid-lsqweemtuk-caseinfoid-fvtmv2oyuw-lsqweemtuk-caseinfoid-fvtmv2oyuw-caseinfoid-fvtmv2oyuw-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-fvtmv2oyuw-caseinfoid-lsqweemtuk-caseinfoid-fvtmv2oyuw-lsqweemtuk-caseinfoid-fvtmv2oyuw-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-field-changed-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-field-changed-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-d2cabf40-bcf7-11ed-b6eb-39368307a439",
+                                "sectionItemType": "field",
+                                "startCol": 1
+                            }
+                        ],
+                        "maxW": 3,
+                        "minH": 1,
+                        "moved": false,
+                        "name": "Hunting Hypothesis - SDO Details",
+                        "static": false,
+                        "w": 2,
+                        "x": 0,
+                        "y": 2
+                    },
+                    {
+                        "description": "A summary of the hunt results including tool usage found, affected hosts and affected users",
+                        "displayType": "CARD",
+                        "h": 2,
+                        "hideName": false,
+                        "i": "caseinfoid-field-changed-caseinfoid-lsqweemtuk-caseinfoid-fvtmv2oyuw-lsqweemtuk-caseinfoid-fvtmv2oyuw-caseinfoid-fvtmv2oyuw-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-fvtmv2oyuw-caseinfoid-lsqweemtuk-caseinfoid-fvtmv2oyuw-lsqweemtuk-caseinfoid-fvtmv2oyuw-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-field-changed-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-lsqweemtuk-field-changed-lsqweemtuk-caseinfoid-lsqweemtuk-caseinfoid-7d826ee0-d1d6-11ed-b353-b1e63aacf41c",
```

Comment:
Any idea what these are?

Reply:
Looks like its when defining section to be displayed as cards